core-bundle/src/DependencyInjection/Compiler/MakeServicesPublicPass.php

```diff
@@ -33,6 +34,7 @@ class MakeServicesPublicPass implements CompilerPassInterface
         'security.logout_url_generator',
         'security.helper',
         'uri_signer',
+        Countries::class,
```

Comment:
Why not just declare the service public in `services.yaml`?

Reply:
I felt that there was something wrong when I wrote this... ☺️
Fixed in 8181b663df625e1d214440d8535d1f10331490ba